### ISSUE TITLE
Studying @ ALX SOFTWARE ENGINEERING PROGRAM

### ISSUE THREAD

**Issue by EMMANUEL-JOTHAM77** [ayyesu/ALX-Software-Engineering-Programme#1]

**Comment by ayyesu** [comment]
Thanks for your comment

**Comment by ayyesu** [comment]
Are you also studying?

**Comment by EMMANUEL-JOTHAM77** [comment]
Yes. I just subscribed to the ALX Software Engineering programme.

**Comment by ayyesu** [comment]
Wow that's great!!
Hoping to be highly equipped after the course

**Comment by Biillyyy** [comment]
Just got here, I'm also a participant guys. It'd be nice yo get along together.

**Comment by ayyesu** [comment]
> Just got here, I'm also a participant guys. It'd be nice yo get along together.

That's a good idea 
So we can share ideas to be able to progress

**Comment by Sanxile** [comment]
Hey are we all studying the same thing here

**Comment by Biillyyy** [comment]
Sanxile, I think so

**Comment by EMMANUEL-JOTHAM77** [comment]
Yes

**Comment by Sanxile** [comment]
> Sanxile, I think so

Cool then work we shall

**Comment by Biillyyy** [comment]
Should I open a Kind of group to add you guys or??

**Comment by EMMANUEL-JOTHAM77** [comment]
Yes, please do.

**Comment by ayyesu** [comment]
> Should I open a Kind of group to add you guys or??

Yes please

**Comment by ayyesu** [comment]
Hello anyone has a Jasper code so I can join?

**Comment by Biillyyy** [comment]
Emmanuel, send me your phone number via Email

**Comment by EMMANUEL-JOTHAM77** [comment]
Just got it 1F7E5E2A

**Comment by EMMANUEL-JOTHAM77** [comment]
You can join us on 1F7E5E2A @Jasper

**Comment by EMMANUEL-JOTHAM77** [comment]
+2348108944560

**Comment by Thomas-tunde** [comment]
> Should I open a Kind of group to add you guys or??

Yes and please drop the link here as well
